Makefile

```diff
@@ -234,16 +233,16 @@ env:
 # of pip don't think CWD is a valid package.
 install: $(DIST_DIR)/natcap.invest%.whl
 	-$(RMDIR) natcap.invest.egg-info
-	$(PIP) install --isolated --upgrade --no-index --only-binary natcap.invest --find-links=dist natcap.invest
+	$(PIP) install --isolated --upgrade --no-index --only-binary natcap.invest --find-links=dist "natcap.invest==$(VERSION)"
```

Comment:
It's not in the diff, but making `VERSION` still requires a direct `setup.py` invocation. Is that okay?

Reply:
I'm curious about this as well, the line is: `VERSION := $(shell $(PYTHON) setup.py --version)`

Reply:
Long-term, probably not!  But short-term, I think this is probably OK enough because `setuptools` isn't really doing a lot of work here since that's delegated to `setuptools_scm` .... though that delegation is, admittedly, done through a convoluted setuptools-managed system of entrypoints and who knows how stable that is.  So we should eventually get away from invoking `setup.py` even for the version string.

Unfortunately, the `setuptools_scm` devs don't have a good way to do this that bypasses `setup.py` entirely _right now_.  There is an active PR in the works (https://github.com/pypa/setuptools_scm/pull/646), which would enable `python -m setuptools_scm` to read from `pyproject.toml` as it should so that's the option I'd prefer to go with, but I don't know when it'll be released.  Assuming this approach is OK with everyone, I've created https://github.com/natcap/invest/issues/764 and assigned it to the InVEST 3.10.1 milestone so we can be sure to revisit this when we can.

Reply:
Okay, thank you. Mainly just wanted to confirm it wasn't an oversight to leave it in for now and it sounds like it certainly is not.